thirdparty/basis_universal/encoder/basisu_comp.cpp

```diff
@@ -28,7 +28,7 @@
 #endif
 
 #if BASISD_SUPPORT_KTX2_ZSTD
-#include "../zstd/zstd.h"
+#include <zstd.h>
```

Comment:
I'll make a patch file for this once I confirm the changes work, and see what upstream thinks about it.

Reply:
There's already an upstream PR for it: https://github.com/BinomialLLC/basis_universal/pull/228